### PR TITLE
Add --additional-gids to runc exec.

### LINKED ISSUE #1306: runc exec: cli flag to specify additional gids
`runc exec` only allows uid and primary gid as command line arguments and doesn't have an option to specify additional gids. You have to pass process.json (-p option) to specify additional gids.

This issue is to enhance runc exec to allow specifying additional gids on the command line as a new argument.

### PR DESCRIPTION
**Pull request by sumitsanghrajka**
This flag allows specifying additional gids for the process.
Without this flag, the user will have to provide process.json which allows additional gids.
Closes #1306

Signed-off-by: Sumit Sanghrajka <sumit.sanghrajka@gmail.com>

### CODE REVIEW

**Comment by GrantSeltzer** [comment]
urfave/cli provides the capability for an `IntSliceFlag` which can be used instead of forcing the user to specify by a comma and then splitting it out.

**Comment by sumitsanghrajka** [comment]
@GrantSeltzer done.

**Comment by GrantSeltzer** [comment]
@sumitsanghrajka Looks like the build is failing because you forgot to sign off on that last commit. Otherwise it looks good, I don't know if the int64 flag is necessary over the int one but I don't see why it'd be an issue other than for consistency.

**Comment by sumitsanghrajka** [comment]
@GrantSeltzer gids can go upto 2^32 value, so int won't work hence using int64.

**Comment by GrantSeltzer** [comment]
@sumitsanghrajka Ah, didn't catch that.

File: exec.go [hunk 1]
```diff
@@ -204,5 +208,8 @@ func getProcess(context *cli.Context, bundle string) (*specs.Process, error) {
 		}
 		p.User.UID = uint32(uid)
 	}
+	for _, gid := range context.Int64Slice("additional-gids") {
+		p.User.AdditionalGids = append(p.User.AdditionalGids, uint32(gid))
```
Comment (cyphar) [inline exec.go]: I would recommend doing a check to make sure `gid` isn't negative.

**Pull request closed without merging**